AIPscan/Aggregator/database_helpers.py

```diff
@@ -306,6 +307,18 @@ def _add_normalization_date(file_id):
         db.session.commit()
 
 
+def _add_premis_object_xml(fs_entry, file_id):
+    """Add string representation of PREMIS Object to File object."""
+    file_ = File.query.get(file_id)
+    for ss in fs_entry.amdsecs[0].subsections:
+        if ss.contents.mdtype == fs_entry.PREMIS_OBJECT:
+            file_.premis_object = etree.tostring(
+                ss.contents.serialize(), pretty_print=True
+            )
+    db.session.commit()
+    return
```

Comment:
a return None is more explicit. Perhaps add return types and inputs in fields works fine either way though.